alinka/widget/containers/main_body/footer/application.py

```diff
@@ -31,7 +31,13 @@ def validate_document_data(self) -> None:
 
     def print_documents(self) -> None:
         self.validate_document_data()
-        generate_and_save_decision(form_data=self.document_data, generate=True)
+        destination_path = QFileDialog.getExistingDirectoryUrl()
```

Comment:
Two things here:
 * did you considered using `getExistingDirectory` instead of `getExistingDirectoryUrl`?
 * can we pass default `Settings.DOCUMENTS_PATH` to start searching directory there instead of `pwd`?